### PR TITLE
Add simple agent workload capture and report

### PR DESCRIPTION
**Pull request by jtaleric**
- Added Agent workload for pod density
- Capture result
- Display result to user

Signed-off-by: Joe Talerico <rook@isovalent.com>

### CODE REVIEW

File: roles/benchmarks/tasks/agent.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    kubectl get nodes -o wide -o jsonpath="{.items[*].status.nodeInfo.kernelVersion}" | wc -w
+  ignore_errors: true
+  register: node_count
+
+- name: Retrieve start time
+  shell: |
+    cat {{archive_dir}}/starttime
+  ignore_errors: true
+  register: start_time
+
```
Comment (jtaleric) [inline roles/benchmarks/tasks/agent.yml:49]: I think a lot of the above could be moved to a common role shared between datapath and agent.

File: roles/benchmarks/tasks/agent.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- name: Setup kubeconfig
+  set_fact:
+    kubeconfig: "{{archive_dir}}/kubeconfig"
+  when: kubeconfig|length < 1
+
+- name: Check for benchmark-operator
+  shell: |
+    export KUBECONFIG={{kubeconfig}}
+    kubectl get pods -n benchmark-operator --show-labels -l control-plane=controller-manager | grep controller-manager
+  register: bmo
```
Comment (learnitall) [inline roles/benchmarks/tasks/agent.yml:11]: There's a check on this return code at the bottom, but why not do a failure condition here first?

File: roles/benchmarks/tasks/agent.yml (hunk 1 above)
Comment (learnitall) [reply roles/benchmarks/tasks/agent.yml:49]: Yeah maybe we could create a metadata role or task or something that can help trigger fact setting like this for other roles.

**Review by learnitall — COMMENTED** [review]
Just one tiny thing, but lgtm!

File: roles/benchmarks/tasks/agent.yml (hunk 2 above)
Comment (jtaleric) [reply roles/benchmarks/tasks/agent.yml:11]: I think we should move this to a common task file to share for all benchmarks. But agree on failing fast.

File: roles/cilium/tasks/gke.yml [hunk 3]
```diff
@@ -27,6 +27,11 @@
   register: cilium_install
   ignore_errors: true
 
+- name: Store Cilium version
```
Comment (learnitall) [inline roles/cilium/tasks/gke.yml:30]: Do we also want to store the `cilium-cli` version?

**Review by learnitall — APPROVED** [review]
LGTM!

File: roles/cilium/tasks/gke.yml (hunk 3 above)
Comment (jtaleric) [reply roles/cilium/tasks/gke.yml:30]: wouldn't hurt.

Comment (jtaleric) [reply roles/cilium/tasks/gke.yml:30]: Added

**Pull request merged**